docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,259 @@
+# Other topics
+
+## Using a DB with concord
+
+Sometimes, when you are developing a command, you may wish to save some data, that will be not deleted across the restarts from your bot, this is where a database comes in. It can save data, and, don't worry, it will not be deleted if you restart your bot.
```

Comment:
"Sometimes, when you are developing a bot command, you may wish to save some data that will be not deleted across bot restarts. This is where a database comes in. It can save data, and, don't worry, it won't be deleted if you restart your bot."

Reply:
Applied, thank you :)